.travis.yml

```diff
@@ -3,10 +3,34 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - 7
   - hhvm
 
+matrix:
+  fast_finish: true
+  allow_failures:
+    - php: 7
+
+env:
+  matrix:
+    - LOWEST_DEPS="" TEST_DEPS=""
+    - LOWEST_DEPS="--prefer-lowest" TEST_DEPS="tests/Sabre/"
```

Comment:
That's not correct, we don't have the `tests/Sabre/` directory. Moreover we don't use the `TEST_DEPS` variable so you can remove it.